example/lib/main.dart

```diff
@@ -32,8 +34,10 @@ class ExamplePage extends StatelessWidget {
         children: <Widget>[
           _createListItem(
             title: "Original color",
-            subtitle: """TinyColor tc = TinyColor(Colors.blue)
-Color c = Colors.blue""",
+            subtitle: """
+TinyColor tc = TinyColor(Colors.blue)
+Color c = Colors.blue
+            """,
```

Comment:
this `”””` ending should have no spaces before it (start of line)

Reply:
True. Could you quickly fix that please?